### PR TITLE
cleo: Add compatibility wrapper for v1

### LINKED ISSUE #215: [BUG]  Archlinux cleo from repository is api incompatible
Recently archlinux upgraded python-cleo to v1.0.0, which is api incompatible with v0.8.1, thus breaking trakt-scrobbler on archlinux (it was upgraded to satisfy dependency of python-poetry)

**Desktop (please complete the following information):**
 - OS and Version: Arch Linux
 - Python Version: 3.10.7
 - Player and Version: mpv 0.34.1-dirty

**To Reproduce**

1. run `trakts` from terminal
2. See error in log file

**Log file**

<details><summary>Click to see log contents</summary> 
<p>

```<!--DO NOT REMOVE THE THREE BACKTICKS. Replace only the next line completely-->
2022-10-09 11:57:57,456 - ERROR - MainThread - __init__ - Unhandled exception
Traceback (most recent call last):
  File "/usr/lib/python3.10/runpy.py", line 196, in _run_module_as_main
    return _run_code(code, main_globals, None,
  File "/usr/lib/python3.10/runpy.py", line 86, in _run_code
    exec(code, run_globals)
  File "/home/mubashshir/Documents/git/trakt-scrobbler/trakt_scrobbler/__main__.py", line 4, in <module>
    from .console import main
  File "/home/mubashshir/Documents/git/trakt-scrobbler/trakt_scrobbler/console.py", line 2, in <module>
    from cleo import Application
ImportError: cannot import name 'Application' from 'cleo' (/usr/lib/python3.10/site-packages/cleo/__init__.py)
2022-10-09 11:57:57,457 - DEBUG - MainThread - notifier - Notifications enabled for categories: exception, misc, scrobble.pause, scrobble.resume, scrobble.start, scrobble.stop, trakt
```
</p>
</details>

### PR DESCRIPTION
**Pull request by ahmubashshir**
Closes #215
Signed-off-by: Mubashshir <ahmubashshir@gmail.com>

### CODE REVIEW

**Comment by ahmubashshir** [comment]
@iamkroot how about this?

File: trakt_scrobbler/_cleo.py [hunk 1]
```diff
@@ -0,0 +1,10 @@
+from cleo import __version__
```
Comment (iamkroot) [inline trakt_scrobbler/_cleo.py:1]: Let's not import it as a bare var. Do something like `from cleo import __version__ as cleo_version`.

**Review by iamkroot — COMMENTED** [review]
Have you verified that all the commands are working fine? I'm not sure how much of the API has changed with v1.

**Comment by ahmubashshir** [comment]
On Sat, Oct 8 2022 at 11:39:31 PM -07:00:00, Krut Patel 
***@***.***> wrote:
> Have you verified that all the commands are working fine? I'm not 
> sure how much of the API has changed with v1.
> 

I'm testing these rn... tested run/start/stop

	Mubashshir

**Comment by ahmubashshir** [comment]
> I'm not sure how much of the API has changed with v1.

Nothing much, mostly they renamed things here and there, used `Enum` instead of constants and dropped dependency on clikit, providing their own `ui` and `io` modules.

**Comment by ahmubashshir** [comment]
> > I'm not sure how much of the API has changed with v1.
> 
> Nothing much, mostly they renamed things here and there, used `Enum` instead of constants and dropped dependency on clikit, providing their own `ui` and `io` modules.

@python-poetry didn't implement cell wrapping in cleo implementation yet...

**Comment by ahmubashshir** [comment]
nah... I'm scrapping this... poetry guys borked cleo so badly that subcommands doesn't work anymore... I'll try something less docs consuming on my end...

**Pull request closed without merging**